### PR TITLE
[KK-0007]: :fix: Fixed the problem with npm start

### PR DESCRIPTION
**Pull request by kinga-kus**
# Description
The problem was with the command npm start. The page wasn't loading and it threw an Error "0308010c:digital envelope routines::unsupported". 
The second problem was with prettier configuration while the web was running.

## Details

I have changed
`"scripts": {
    "start": "react-scripts --openssl-legacy-provider start",
  }`
in the package.json and then upgrade react-scripts using:
#### Uninstall react-scripts: 
`npm uninstall react-scripts` 
#### and then installed again 
`npm install react-scripts`

And deleted and installed again node_modules and package-lock.json.

The second problem fixed installing eslint-config-prettier 
`npm install --save-dev --save-exact prettier`

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+    "extends": ["prettier"],
+    "plugins": ["prettier"],
+    "rules": {
+      "prettier/prettier": 2 // Means error
```
Comment (adam-kostuch) [inline .eslintrc:5]: What do you mean by `Means error`?  🤔

**Comment by adam-kostuch** [comment]
Also there's no need to add `:fix:` to the PR title, there are to many `fix`'alike words there 🤣

File: .eslintrc (hunk 1 above)
Comment (kinga-kus) [reply .eslintrc:5]: It showed when i have installed the prettier, should I delete this?

**Review by adam-kostuch — APPROVED** [review]

File: .eslintrc (hunk 1 above)
Comment (adam-kostuch) [reply .eslintrc:5]: Nah, there's no need for that, you can go and merge it!

**Comment by kinga-kus** [comment]
But You have told me to write this

**Pull request merged**